docs/dev/table/streaming/time_attributes.md

```diff
@@ -317,9 +323,15 @@ class UserActionSource extends StreamTableSource[Row] with DefinedRowtimeAttribu
 		stream
 	}
 
-	override def getRowtimeAttribute = {
+	override def getRowtimeAttributeDescriptors: util.List[RowtimeAttributeDescriptor] = {
 		// Mark the "UserActionTime" attribute as event-time attribute.
-		"UserActionTime"
+		// here we create one attribute descriptor of "UserActionTime"
+		val rowtimeAttrDescr = new RowtimeAttributeDescriptor(
+		    "UserActionTime",
```

Comment:
Same as above.

Reply:
this too
![image](https://user-images.githubusercontent.com/13216107/51025094-2598f580-15c6-11e9-8d8c-2c3249ec5a77.png)

Reply:
@KarmaGYZ pls tell me if there is anything need be improve, Thanks.

Reply:
@JaryZhen I still prefer to use tab. Besides, I think you should push this commit the master branch.